app/src/main/java/woowacourse/paint/MainViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package woowacourse.paint
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import woowacourse.paint.model.ColorBox
+
+class MainViewModel : ViewModel() {
+    private val _colors = MutableLiveData<List<ColorBox>>()
+    val colors: LiveData<List<ColorBox>>
+        get() = _colors
+
+    init {
+        setColors()
+    }
+
+    private fun setColors() {
+        _colors.value = paintColors
+    }
```

Comment:
```suggestion
    private val _colors = MutableLiveData(paintColors)
    val colors: LiveData<List<ColorBox>>
        get() = _colors
```

최초 한번만 초기화 하는 용도라면 다음과 같이 변경해보는 것은 어떨까요??